### ISSUE TITLE
String dialog data breaks in Chrome

### ISSUE THREAD

**Issue by PifLeChienBis** [likeastore/ngDialog#420]
When opening a dialog that has string data, ngDialog tries to assign the id property of data (in this case the string). Although IE (11) and Firefox (38.5) seem to be cool with that, in strict mode Chrome will raise an error : by default, properties are readonly (even though a property definition can't be fetched in my case) and Chrome will raise the error.

From ngDialog.js commit ac81282c7d43e996b289a43a6051072321754102, line 515 : 

```javascript
if (options.data && angular.isString(options.data)) {
  var firstLetter = options.data.replace(/^\s*/, '')[0];
  scope.ngDialogData = (firstLetter === '{' || firstLetter === '[') ? angular.fromJson(options.data) : options.data;
  scope.ngDialogData.ngDialogId = dialogID;
} else if (options.data && angular.isObject(options.data)) {
  scope.ngDialogData = options.data;
  scope.ngDialogData.ngDialogId = dialogID;
}
```

Here's a [jsFiddle](https://jsfiddle.net/8e3m3rmq/1/) of the error.

**Comment by BarryWaluszko** [comment]
fixed in #421 
ngDialog v.0.5.7

**Comment by faceleg** [comment]
Thanks @BarryWaluszko
